modules/market/src/main/java/com/opengamma/strata/market/curve/definition/CurveGroupEntry.java

```diff
@@ -69,6 +70,24 @@
   @PropertyDefinition(validate = "notNull")
   private final ImmutableSet<OvernightIndex> overnightIndices;
 
+  //-------------------------------------------------------------------------
+  /**
+   * Merges the specified entry with this entry.
```

Comment:
Probably worth mentioning that a new value is returned and this value and the input aren't modified.